plugins/modules/database.py

```diff
@@ -34,9 +34,9 @@
         Only used if database is being created.
     type: str
     required: false
-  owner_name:
+  owner:
```

Comment:
is this intended? seems like a breaking change

Reply:
Yeah, but at v0 I'm just rolling with a few of them as they come up.